_posts/2025-11-10-introducing-jarl/introducing-jarl.Rmd

```diff
@@ -63,34 +87,45 @@ The latter adds a lightbulb button next to rule violations, allowing you to sele
   <source src="demos/quick-fixes.webm" type="video/mp4">
 </video>
 
+<br>
+
+In the future, those extensions could have a "Fix on save" feature similar to the "Format on save" functionality provided by Air.
+
 
 ## Configuring Jarl
 
 By default, Jarl will report violations for almost of its rules.
 It is possible to configure its behavior using a configuration file named `jarl.toml`.
 In particular, in this file, you can specify:
 
-* the rules you want to apply
-* the files to include or exclude
-* the rules for which you want to apply automatic fixes
+* the rules you want to apply,
```

Comment:
Mention exception comments that you can add in files? Just one sentence.

Reply:
Thanks, but I'd prefer people to go to the docs for details like this.